File: Java/conditionals/pollution_index_calc.java
package br.com.geekuniversity.secao06;

import java.util.Scanner;

public class Exercicio09 {

	public static void main(String[] args) {
		//Declaração
		float indice;
		Scanner teclado = new Scanner(System.in);
		
		//Entrada
		System.out.println("Informe o índice de poluição: ");
		indice = teclado.nextFloat();
		
		//Processamento
		if (indice >= 0.5) {
			//Saída
			System.out.println("Todos os grupos devem suspender as atividades!");
		} else if (indice >= 0.4) {
			//Saída
			System.out.println("Grupo 1 e Grupo 2 - Suspender atividades.");
		} else if (indice >= 0.3) {
			//Saída
			System.out.println("Grupo 1 - Suspender as atividades.");
		} else {
			//Saída
			System.out.println("Aceitável.");
		}
		teclado.close();
	}
}

File: Python/conditionals/pollution_index_calc.py
# coding=utf8
# Entrada
indice = float (input("Informe o índice de poluição: "))

if indice >= 0.5:
    print("Todos os Grupos devem Suspender suas atividades!")
elif indice >= 0.4:
    print("Grupo 1 e Grupo 2 - Suspender suas atividades")
elif indice >= 0.3:
    print("Grupo 1 - Suspender suas atividades")
else:
    print("Aceitável")